tensorboard/plugins/image/tf_image_dashboard/tf-image-loader.html

```diff
@@ -58,31 +58,24 @@
       </template>
     </tf-card-heading>
     <div id="main-image-container"></div>
-    <div style="display: flex; flex-direction: row;">
-      <paper-icon-button
-        selected$="[[_expanded]]"
-        icon="fullscreen"
-        on-tap="_toggleExpanded"
-      ></paper-icon-button>
-      <paper-icon-button
-        icon="aspect-ratio"
-        on-tap="_showActualSize"
-        title="Show the image at its true pixel size"
-      ></paper-icon-button>
-    </div>
 
     <style>
       :host {
         display: block;
-        width: 330px;
+        width: 332px;
```

Comment:
The previous width was actually awry and excluded the border, although this make a difference in practice because it did not affect positioning.